communication/calendar-guidelines.md

```diff
@@ -95,6 +100,54 @@ will prevent you from getting updates like a canceled meeting. Join the main
 
 Accept the invite from the sender and you'll have the updates.
 
+### Calendar event template
+
+In case you are creating a calendar event and wondering what to include in it, you can refer to the following template
+
+#### Title
+
+```
+[SIG-<name>] <meeting title>
+or
+[WG-<name>] <meeting title>
+or
+[User Group-<name>]
+```
+#### Description
+
+```
+<One line intro about the SIG/ WG. Can include github links>
+
+Time: <Weekly/ biweekly/... or is the meeting for a fixed time and till when?>
+
+Meeting Notes & Agenda: <Hyperlink to the meeting notes doc>
+
+Zoom link: <Zoom invite link. Preferably both app and web client links>
+
```

Comment:
<Dial-in phone number and instructions>

Reply:
Is there supposed to something here? I don't find any change

Reply:
Is there supposed to be something here? I don't find any suggestion/ change.

Reply:
Oops, ignore this one :eyes: